R/utils.R

```diff
@@ -18,7 +18,7 @@ replace_assignment = function(exp) {
 }
 
 ## mask comments to cheat R
-mask_comments = function(x, width, keep.blank.line, wrap = TRUE, spaces) {
+mask_comments = function(x, keep.blank.line, spaces) {
   d = utils::getParseData(parse_source(x))
   if (nrow(d) == 0 || (n <- sum(d$terminal)) == 0) return(x)
```

Comment:
This `n <- sum(d$terminal` part in side the condition is stinky in more than one ways. Change?

Reply:
I often use this style myself (assignment and condition in one expression). Do you mean this?

```suggestion
  if (nrow(d) == 0) return(x)
  n = sum(d$terminal)
  if (n == 0) return(x)
```

Usually I prefer [relatively dense code](https://yihui.org/cn/2017/01/blank-area/), so I don't need to scroll up and down frequently.